### PR TITLE
Use archivematica-browse-helpers

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

File: app/services/sip_arrange.service.js [hunk 1]
```diff
@@ -59,36 +60,13 @@ factory('SipArrange', ['Restangular', function(Restangular) {
...
 
-    var format_entries = data => {
-      return data.entries.map(element => {
-        var child = {
-          title: element,
-          path: parent ? parent.path + '/' + element : element,
-          parent: parent,
-          display: true,
-          properties: data.properties[element],
-          type: 'arrange_entry',
```
Comment (Hwesta) [inline app/services/sip_arrange.service.js:70]: `type: 'arrange_entry'` doesn't appear in browse-helpers.js.  Is this going to be a problem?

File: app/services/sip_arrange.service.js [hunk 2]
```diff
@@ -59,36 +60,13 @@ factory('SipArrange', ['Restangular', function(Restangular) {
       });
     };
 
-    var format_entries = data => {
-      return data.entries.map(element => {
-        var child = {
-          title: element,
-          path: parent ? parent.path + '/' + element : element,
```
Comment (Hwesta) [inline app/services/sip_arrange.service.js:66]: In browse-helpers, if the parent_path is null, then '/' is prepended to element.  Does this matter?

File: app/services/sip_arrange.service.js (hunk 1 above)
Comment (mistydemeo) [inline app/services/sip_arrange.service.js:70]: Good catch; updated to append that to the values returned by `format_entries`.

File: app/services/archivesspace.service.js [hunk 3]
```diff
@@ -96,7 +57,7 @@ factory('ArchivesSpace', ['Restangular', function(Restangular) {
       },
       list_digital_object_component_contents: function(id, component_id) {
         var id_fragment = id_to_urlsafe(id);
-        return ArchivesSpace.one(id_fragment).one('digital_object_components').one(String(component_id)).one('files').get().then(decode_entry_response).then(format_results);
+        return ArchivesSpace.one(id_fragment).one('digital_object_components').one(String(component_id)).one('files').get().then(decode_browse_response).then(format_entries);
```
Comment (Hwesta) [inline app/services/archivesspace.service.js]: Are parent_path and parent getting passed to format_entries, or are they supposed to be undefined/null?

Comment (mistydemeo) [inline app/services/archivesspace.service.js:60]: I'm pretty sure this is broken; it hasn't been used in awhile. I'll remove in a followup PR.

File: app/services/sip_arrange.service.js (hunk 2 above)
Comment (mistydemeo) [inline app/services/sip_arrange.service.js:66]: Fixed in https://github.com/artefactual-labs/archivematica-browse-helpers/pull/1.

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**